handler.go

```diff
@@ -43,6 +44,7 @@ import (
 
 	"unicode"
 
+	// Allow building Pilosa without the web UI.
```

Comment:
I was just addressing a golint warning, and thought it would be helpful to comment why these blank imports exist